resources/js/Pages/Home.vue

```diff
@@ -29,3 +29,9 @@ provide('isLoginRef', isLogin)
     <Footer />
   </div>
 </template>
+
+<style scoped>
+html {
+  overflow-y: scroll;
+}
+</style>
```

Comment:
Why here? Not in layout?

Reply:
This is the only page where it is going to be used, to fix an edge case where page resizes itself when search is opened